specification/components/r4/schemas/eRS-ServiceRequest.yaml

```diff
@@ -153,26 +71,40 @@ properties:
             required:
               - system
               - code
-              - display
             properties:
               system:
                 type: string
                 enum:
-                  - 'http://snomed.info/sct'
+                  - 'https://fhir.nhs.uk/CodeSystem/message-category-servicerequest'
               code:
                 type: string
-                example: '820641000000100'
-              display:
-                type: string
-                example: 'Choose and book advice and guidance request'
-  priority:
-    type: string
-    enum:
-      - routine
-      - urgent
-      - asap
-      - stat
-    example: 'asap'
+                enum:
+                  - referral
+                  - request
+                example: 'referral'
```

Comment:
Looks like the formatting is a bit off here, I think this should be just one level below `priority` above?

Reply:
The priority chunk above was removed - this enum  is for the code in category:

```
  category:
    type: array
    items:
      type: object
      required:
        - coding
      properties:
        coding:
          type: array
          items:
            type: object
            required:
              - system
              - code
            properties:
              system:
                type: string
                enum:
                  - 'https://fhir.nhs.uk/CodeSystem/message-category-servicerequest'
              code:
                type: string
                enum:
                  - referral
                  - request
                example: 'referral'
```